app/views/devise/passwords/new.html.haml

```diff
@@ -0,0 +1,14 @@
+.card-panel.auth-card
+  .beer-logo-container
+    =image_tag('Beer-icon.png', class: 'beer-logo')
+    %h3 Forgot your password?
+    = form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f|
+      - devise_error_messages!
+      .input-field
+        = f.label :email
+        %br/
+        = f.email_field :email, autofocus: true, autocomplete: "email"
+      .actions
+        = f.submit "Reset password", class: 'btn blue waves-effect waves-light submit-button'
```

Comment:
Use single quotes `'` instead of `"` in all the files for convention, what do you think?